lib/chef/provider/aws_security_group.rb

```diff
@@ -93,14 +93,14 @@ def update_inbound_rules(sg, vpc)
 
       authorize: proc do |port_range, protocol, actors|
         names = actors.map { |a| a.is_a?(Hash) ? a[:group_id] : a }
-        converge_by "authorize #{names.join(', ')} to send traffic to group #{new_resource.name} (#{sg.id}) on port_range #{port_range} with protocol #{protocol}" do
+        converge_by "authorize #{names.join(', ')} to send traffic to group #{new_resource.name} (#{sg.id}) on port_range #{port_range || 'nil'} with protocol #{protocol || 'nil'}" do
```

Comment:
If you want it to do a really good job printing, just use `port_range.inspect` and it will do this for you.